sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTableCell.cs

```diff
@@ -14,7 +14,7 @@ internal FormTableCell(DataTableCell_internal dataTableCell, IReadOnlyList<ReadR
         {
             ColumnIndex = dataTableCell.ColumnIndex;
             ColumnSpan = dataTableCell.ColumnSpan ?? 1;
-            Confidence = dataTableCell.Confidence;
+            Confidence = dataTableCell.Confidence == 0 ? Constants.DefaultConfidenceValue : dataTableCell.Confidence;
```

Comment:
According to the swagger file, the confidence here is marked as required. Is there any reason for overwriting the value when it's 0 instead of accepting it? Is there a scenario I'm missing?

Reply:
trying this again.
Although it is required in the Swagger, the service returns null. while they work on that, we decided to always put as a default 1.0f value.

Because of how we do our serialization, we assign Confidence to default, which is 0f.
See https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/Generated/Models/DataTableCell_internal.Serialization.cs#L24

Reply:
Got it!